src/content/ethereum-blockchain-getting-started/index.md

```diff
@@ -0,0 +1,188 @@
+   ```
+
+1. Make a directory for your new project and navigate to your new directory:
+
+   ```bash
+   mkdir truffle-example
+   ```
+
+   ```bash
+   cd truffle-example
+   ```
+
+1. Unbox a truffle box. We're going to use one called drizzle. There are many other boxes available <a href="https://www.trufflesuite.com/boxes" target="_blank">here</a>
+
+   ```bash
+   truffle unbox drizzle
+   ```
+
+1. Let's see what we have. Open the contents in your text editor of choice. For example, I'm using vs-code `code` command:
+
+   ```bash
+   code .
+   ```
+
+   Inside of our project, we should have an app/ directory. This is essentially a create-react-app that has additional features that allows it to integrate with a blockchain, either running on our computer or on the ethereum mainnet. We also have a contracts/ directory which contains our 4 example contracts. Notice the file extensions are .sol for solidity. Next, we will see a directory for migrations/ which allows us to deploy changes to our smart contracts. We also have a test/ directory which contains our smart contract test files. Finally we have a package-lock.json, package.json, and truffle-config.js files.
+
+   <img src="./images/truffle_unbox_drizzle.png">
+
+1. Now lets compile our contracts and try to run our app.
+
+   ```bash
+   truffle compile
+   ```
+
+1. Open a new terminal window, navigate into our React app and lets start it:
+
+   ```bash
+   cd app
+   ```
+
+   ```bash
+   npm start
+   ```
+
+   If successful, your app should open in a new browser tab and you should see a Loading... message. This is because the app is trying to connect to a blockchain but unable to. To resolve this, we will need to deploy our contracts to a local blockchain using <a href='https://www.npmjs.com/package/ganache-cli' target='_blank'>ganache-cli</a>
+
+## Use ganache-cli to run a local blockchain
+
+1. Open a new terminal window. We're going to install ganache-cli globally:
+
+   ```bash
+   npm install -g ganache-cli
+   ```
+
+1. Run ganache-cli to get a local ethereum blockchain running on your computer.
+
+   ```bash
+   ganache-cli
+   ```
+
+   when successful, this command will provide you with 10 available accounts and corresponding private keys, each account with 100 ETH. Will also provide a mnemonic or 12-word phrase. The blockchain will listen on port 8545.
+
+1. Let's copy this 12-word mnemonic and create a command to run our blockchain as deterministic and mnemonic:
+
+   in our code editor, we are going to edit our package.json inside our truffle-example directory. We are going to create a command called `ganache` to run our blockchain and persist it to a db. In this example, I'm using the mnemonic: 'picture tray motion shuffle action fun puzzle velvet sample trip reject beef' but you can use the mnemonic that was generated by your ganache-cli or another mnemonic as you please. In order for our data to persist, we will need to run ganache-cli with the following flags:
```

Comment:
I think the formatting is a little off in this block. Might want to not have this one intended

Reply:
I think I'm okay with this being indented. This one is a long step but everything indented below is part of step 3.